hs/Database/CourseInsertion.hs

```diff
@@ -2,20 +2,17 @@
     OverloadedStrings, TypeFamilies #-}
 
 {-|
-Description: Odd miscellany of functions related to the database.
+Description: Functions that insert/update course information in the database.
 
 This module contains a bunch of functions related to inserting information
 into the database. These functions are used as helpers for the WebParsing module.
-
-TODO: This module should be renamed, possibly put in a different location,
-and/or split up.
 -}
 
-module Database.JsonParser (insertCourse,
-                    insertLec,
-                    insertTut,
-                    setTutEnrol,
-                    setPracEnrol) where
+module Database.CourseInsertion (insertCourse,
```

Comment:
Similar to my comment on the other branch, I'd like to see `insertCourse` on a new line, indented 4 spaces